### DISCUSSION TITLE
Block level filtering within the current page (Roam style)

### DISCUSSION THREAD

**Discussion by tonyxiao** [vslinko/obsidian-zoom#85] [Ideas]
Roam has the ability for you to filter the current list within a page by a specific backlink or tag. I am really missing it and would love to see it in Obsidian. Is that possible to do via the outliner? I'd be happy to contribute code to make this possible if someone can give me some pointers.

This might be slightly out of scope for the obsidian zoom plugin. However I this is the closest plugin I could find that could possibly implement such a feature given that it already has ability to "zoom" into a subsection of document.

**Comment by vslinko** [comment]
ref: https://github.com/vslinko/obsidian-outliner/discussions/400
